GenericsHomework/GenericsHomework.csproj

```
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<RootNamespace>GenericsHomework_</RootNamespace>
```

Adding this line should satisfy the first requirement to make it a "class library".

```suggestion
		<RootNamespace>GenericsHomework_</RootNamespace>
		<OutputType>Library</OutputType>
```

---

Thank you.